app/main/lib/similarity.py

```diff
@@ -119,3 +135,7 @@ def get_similar_items(item, similarity_type):
   app.logger.info(f"[Alegre Similarity] [Item {item}, Similarity type: {similarity_type}] response for search was {response}")
   return response
 
+def callback_get_similar_items(item, similarity_type):
```

Comment:
is item here the url for the audio item,  or the fingerprint vector we are searching with?

Reply:
At this point in the code, we have the vector from presto in hand now

Reply:
Ok, lets make that clear.  maybe arg is `item_vector`?  validate assumptions about structure before passing to audio model? (i.e. if item really is a numeric vector?)

Reply:
I'm tossing in an assertion routine - please take a look at it in the latest push to this PR and let me know if this satisfies what you're looking for.